integrated_channels/degreed/exporters/course_metadata.py

```diff
@@ -185,9 +186,13 @@ def transform_cost_type(self, course_run):
 
     def transform_title(self, course_run):
         """
-        Return the transformed version of the course title, as well as the locale.
+        Return the transformed version of the course title. For all instructor-paced courses
+        also include the start date to distinguish multiple runs of the same course (ENT-782)
         """
-        return course_run.get('title') or ''
+        title = course_run.get('title') or ''
+        if course_run.get('pacing_type') == 'instructor_paced' and course_run.get('start'):
+            title += ' (Starts: {:%B %Y})'.format(parse_lms_api_datetime(course_run.get('start')))
+        return title
```

Comment:
```
title = course_run.get('title') or ''
course_run_start = parse_lms_api_datetime(course_run.get('start'))
if course_run_start and course_run.get('pacing_type'):
    title += ' (Starts: {:%B %Y})'.format(course_run_start)
return title
```

Reply:
Make sure that `parse_lms_api_datetime` can handle `None`.

Reply:
course_run.get('start') is guarded in the if statement so we shouldn't get `None`

Reply:
Well, not if you do the refactor I suggested above for readability's sake and to remove the duplicate call to `course_run.get`.